apps/Doom 3/install

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+# fix broken install due to @techcoder20
+sudo apt purge -y sdl2-image sdl2-mixer sdl2-ttf || true
```

Comment:
What would happen if these packages were not remove? Would there be conflicts?

Reply:
these packages don't exist in repos. they were custom made during the old install script and cause conflicts with official repo packages (due to overwriting files)

this is the main reason for the re-write

Reply:
> What would happen if these packages were not removed? Would there be conflicts?

oh I think I misread your question. yes they must be purged. my intention here was just to not error if the packages happen to not be installed (like on a clean system) let me fix this